projects/packages/search/package.json

```diff
@@ -0,0 +1,124 @@
+{
+	"private": true,
+	"name": "@automattic/jetpack-search",
+	"version": "0.1.0-alpha",
+	"description": "Search package",
+	"homepage": "https://jetpack.com",
+	"bugs": {
+		"url": "https://github.com/Automattic/jetpack/issues"
+	},
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/Automattic/jetpack.git"
+	},
+	"license": "GPL-2.0-or-later",
+	"author": "Automattic",
+	"scripts": {
+		"build": "echo 'Not implemented.",
+		"build-js": "echo 'Not implemented.",
+		"build-production": "echo 'Not implemented.",
+		"build-production-js": "echo 'Not implemented.",
+		"clean": "true",
+		"distclean": "rm -rf node_modules && pnpm run clean",
+		"install-if-deps-outdated": "pnpm install --no-prod --frozen-lockfile"
+	},
+	"dependencies": {
```

Comment:
You don't seem to actually be using all of these dependencies, for example I don't see any reference to gulp in this package but you're pulling in a ton of gulp stuff.

Reply:
@anomiex thanks for looking at this, the webpack and jest config are still in process. Will clean things up when splitting the real PRs. Thanks.